en/download_button.ftl

```diff
@@ -55,3 +55,13 @@ download-firefox-esr = Download { -brand-name-firefox-esr }
 download-firefox-esr-32 = Download { -brand-name-firefox-esr } 32-bit
 download-firefox-esr-64 = Download { -brand-name-firefox-esr } 64-bit
 download-a-different-build = Download a different build
+
+## Linux
+
+download-button-linux-32 = Download { -brand-name-linux } 32-bit
```

Comment:
Too late now, but we should have remove the brand name for Linux here.

Reply:
Sorry, I saw it had been approved so I merged. I'll get a fix in.

Reply:
My bad!! It didn't take long to forget what we have agreed on.